wormhole-connect-loader/src/types.ts

```diff
@@ -72,4 +72,29 @@ export interface WormholeConnectConfig {
   bridgeDefaults?: BridgeDefaults;
   routes?: string[];
   pageHeader?: string;
+  pageSubHeader?: string;
+  moreTokens?: MoreTokenConfig;
+  extraNetworks?: ExtraChainConfig;
 }
+
+export type MoreTokenConfig = {
```

Comment:
Should the `target` setting also be on this config?

Reply:
good call, addresed